webui/react/src/components/kit/Popover.tsx

```diff
@@ -0,0 +1,27 @@
+import { Popover as AntdPopover } from 'antd';
+import React, { ReactNode } from 'react';
+
+export interface Props {
+  content: ReactNode;
+  placement?:
+    | 'top'
+    | 'left'
+    | 'right'
+    | 'bottom'
+    | 'topLeft'
+    | 'topRight'
+    | 'bottomLeft'
+    | 'bottomRight'
+    | 'leftTop'
+    | 'leftBottom'
+    | 'rightTop'
+    | 'rightBottom';
+  trigger?: 'hover' | 'focus' | 'click' | 'contextMenu' | Array<string>;
+  children?: ReactNode;
+}
+
+const Popover: React.FC<Props> = ({ placement = 'bottom', ...props }: Props) => {
```

Comment:
trigger on click by default?

Reply:
just thinking that this would typically be used to show something larger or more complex than a tooltip, like the menus in the new experiment list. so those would typically be shown on click instead of hover. but this is just a suggestion, default on hover is probably fine too, as long as we have the `trigger` prop.